dlrover/trainer/examples/deepfm/run.sh

```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 rm -rf log* eval/ export/  checkpoint  model.ckpt* events.out* -rf graph.pbtxt
-python -m dlrover.trainer --platform=local --conf=deepfm_conf.TrainConf --ps_num=1 --worker_num=1  
+python -m dlrover.trainer --platform=local_kubernetes --conf=deepfm_conf.TrainConf --ps_num=1 --worker_num=1  
```

Comment:
What's mean "local_kubernetes"?

Reply:
Currently, `dlrover.trainer` supports both `ray` and `kubernetes` by `kubernetes_worker` and `ray_worker`.  In order to debug without submitting to real ray cluster and kubernetes cluster, `dlrover.trainer` are designed to provide the ability to mock lunching the training job by process scheduler. Local_kubernetes means that the job is lunched by process sheduler and try to mock behavior of kubernetes cluster.